src/typings.d.ts

```diff
@@ -37,6 +37,9 @@ declare let ANT_DESIGN_PRO_ONLY_DO_NOT_USE_IN_YOUR_PRODUCTION: 'site' | undefine
 
 declare const REACT_APP_ENV: 'test' | 'dev' | 'pre' | false;
 
+declare let ADMIN_API: '';
```

Comment:
🤔

Reply:
keep getting config data from config file is a simple way in my opinion.

Reply:
This is only used to declare constant types, not specific values.

Reply:
then could you please add a constant.ts file?

Reply:
Oh, it's only used to declare a type. like `declare let ADMIN_API: string;`
It will not be compiled into code for IDE use only.